sgx/panic/src/lib.rs

```diff
@@ -9,6 +9,7 @@
 //
 // Alloc feature must be enabled to get APIs associated to catching panics
 // and rethrowing them, because in Rust those APIs use the Box type.
+#![allow(internal_features)]
```

Comment:
This is kind of an annoying lint. It barks on the use of `feature(lang_items)` but that's the only way to the `eh_personality` that is recommended for customizing panic behavior...